force-app/main/default/classes/ClassesObjectsMethods.cls

```diff
@@ -252,11 +290,15 @@ public with sharing class ClassesObjectsMethods {
      */
     public static String createAndGetBookDetails() {
         // Create a Book object with title "A Brief History of Time" and author "Stephen Hawking"
+        Book myBook = new Book('A Brief History of Time', 'Stephen Hawking');
+        
 
         // Call the getBookDetails method on the book object to get the details of the book
+        String details = myBook.getBookDetails();
+
 
         // Return the details of the book
-        return null; // Replace null with the variable you used to store the result
+        return details; // Replace null with the variable you used to store the result
```

Comment:
Good. As an example of how you can be more concise. This method returns a string and getBookDetails returns a String.  It takes time to see of that though. 

```
  public static String createAndGetBookDetails() {
        return new Book('A Brief History of Time', 'Stephen Hawking').getBookDetails();
    }
 ```